.travis.yml

```diff
@@ -4,8 +4,11 @@ node_js: 12
 branches:
   only:
   - master
+before_install:
+  - curl -L https://unpkg.com/@pnpm/self-installer | node
```

Comment:
Why not use `npm i pnpm` here too?

Reply:
That does make more sense. I was just following their CI guide, so I'm not sure why they don't recommend that.

Reply:
Fixed in 7218d43af7e5f3b584ad09bb2af3e38c8139337d